<launch>
<!--
Launch file for running the multirobot simulation on gazebo
-->
<include file="$(find gazebo_ros)/launch/empty_world.launch">
<arg name="use_sim_time" value="true"/>
<arg name="debug" value="false"/>
<arg name="gui" value="true"/>
<arg name="world_name" value="$(find hunter2_explorer)/world/city.world"/>
</include>

<!-- spawn robot 1-->
<group ns="/robot_1">
	<include file="$(find hunter2_explorer)/launch/include/robot_new.launch">
		<arg name="robot_name" value="robot_1"/>
		<arg name="init_pose" value="-x -3.0 -y 0.0 -z 0.5"/>
	</include>
</group>

<!-- spawn robot 2-->
<group ns="/robot_2">
	<include file="$(find hunter2_explorer)/launch/include/robot_new.launch">
		<arg name="robot_name" value="robot_2"/>
		<arg name="init_pose" value="-x -0.0 -y 0.0 -z 0.5"/>
	</include>
</group>

<!-- spawn robot 3-->
<group ns="/robot_3">
	<include file="$(find hunter2_explorer)/launch/include/robot_new.launch">
		<arg name="robot_name" value="robot_3"/>
		<arg name="init_pose" value="-x 3 -y -0.0 -z 0.5"/>
	</include>
</group>

<!-- run RViz node (visualization) -->
<node pkg="rviz" type="rviz" name="rviz" args="-d $(find hunter2_explorer)/launch/include/rviz_config/three_hunters.rviz">
<remap from="move_base_simple/goal" to="robot_1/move_base_simple/goal"/>
</node>

</launch>